### PR TITLE
Make package compatible with Postgres

### PR DESCRIPTION
**Pull request by shippy**
## Description & motivation

See #69.

## Checklist
- [x] I have verified that these changes work locally
- [x] I have updated the README.md (if applicable)
- [ ] I have added tests & descriptions to my models (and macros if applicable)

## Request for help

I don't know how to create the integration connection to Postgres. I did the cargo cult thing - i.e., I created another sample profile in `integration_tests/ci/sample.profile.yml` and added another section to `integration_tests/Makefile` (+ the BQ section, since that presumably should have been there), but presumably you have to add the environment variables with the credentials, right?

### CODE REVIEW

**Comment by shippy** [comment]
(Pretty sure tests are failing because I added a non-existent target to the `Makefile`, but for obvious reasons I can't see details)

**Comment by shippy** [comment]
Latest commit finalizes the Postgres workaround, so the PR should now be ready for review.

File: packages.yml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 packages:
  - package: dbt-labs/dbt_utils
-   version: [">=0.8.0", "<0.9.0"]
+   version: [">=0.4.0", "<0.9.0"]
```
Comment (jasnonaz) [inline packages.yml]: Hmm what was the reasoning behind dropping this down? We have a required version of dbt v1.0 in this project so any utils below 0.8.0 will fail

File: models/sessionization/segment_web_sessions.sql [hunk 2]
```diff
@@ -31,7 +31,7 @@ with sessions as (
     select * from {{ref('segment_web_sessions__stitched')}}
 
     {% if is_incremental() %}
-    where cast(session_start_tstamp as datetime) > {{sessionization_cutoff}}
+    where cast(session_start_tstamp as {% if target.type == "postgres" -%} timestamp {% else -%} datetime {%- endif -%}) > {{sessionization_cutoff}}
```
Comment (jasnonaz) [inline models/sessionization/segment_web_sessions.sql:34]: @joellabes and I were doing some noodling on this and think that we might be able to replace these with the utils [cross db timestamp macro](https://github.com/dbt-labs/dbt-utils/blob/main/macros/cross_db_utils/datatypes.sql#L25-L38). Mind taking a look and seeing if that does what we need here?

**Review by jasnonaz — APPROVED** [review]
Good to merge! Thank you for submitting this + for your changes. At some point might be worth cleaning up the timestamps to be off the utils cross db timestamp function.

File: packages.yml (hunk 1 above)
Comment (shippy) [reply packages.yml]: This was before i rebased off upstream, to make things work locally - latest version should have the same version requirements

File: models/sessionization/segment_web_sessions.sql (hunk 2 above)
Comment (shippy) [reply models/sessionization/segment_web_sessions.sql:34]: Ah yes, that's a much better approach! Will take a look

File: packages.yml (hunk 1 above)
Comment (shippy) [reply packages.yml]: (as in, HEAD of this branch would have reverted this change)

**Pull request merged**